statsmodels/genmod/generalized_linear_model.py

```diff
@@ -475,6 +475,27 @@ def score_obs(self, params, scale=None):
         score_factor = self.score_factor(params, scale=scale)
         return score_factor[:, None] * self.exog
 
+    def score_obs(self, params, scale=None):
```

Comment:
If you must define `_score_obs`, then in this class just alias `score_obs = _score_obs`